### PR TITLE
Database models

### PR DESCRIPTION
**Pull request by TGZach**
Put in changes from last meeting

### CODE REVIEW

File: db/migrate/20191202001138_create_games.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateGames < ActiveRecord::Migration[5.2]
+  def change
+    create_table :games do |t|
+      t.integer :game_id
+      t.datetime :start_date
+      t.datetime :end_date
```
Comment (vpang79) [inline db/migrate/20191202001138_create_games.rb]: start date and end date entries are redundant. ruby databases automatically keep track of creation date/times and update date/times.

File: app/models/move.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Move < ApplicationRecord
```
Comment (AustinDodd) [inline app/models/move.rb]: Didn't we decide to do away with Moves and we were just going to have User, Game, and Piece? I could be wrong, but I thought that's what we talked about the meeting before last.

File: app/models/piece.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Piece < ApplicationRecord
+  has_many :moves
```
Comment (AustinDodd) [inline app/models/piece.rb]: This will need to be removed if we are in fact not going with Moves.

File: app/models/game.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class Game < ApplicationRecord
+  belongs_to :user
+  has_many :moves
```
Comment (AustinDodd) [inline app/models/game.rb]: This will need to be removed if we are in fact not going with Moves.

File: db/migrate/20191202002030_create_moves.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+class CreateMoves < ActiveRecord::Migration[5.2]
```
Comment (AustinDodd) [inline db/migrate/20191202002030_create_moves.rb]: Again about the Moves.

**Pull request merged**